### PR TITLE
Update action.yml with Auth Token

### PR DESCRIPTION
**Pull request by sbrij001**
removed api key and api token and replaced with auth token

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by sbrij001** [comment]
@bnb Can you review this PR?

File: action.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ name: 'Twilio SMS'
 author: 'Twilio Labs'
 description: 'Send an SMS from GitHub Actions using Twilio Programmable SMS'
 inputs:
-  fromPhoneNumber:
+  FROM_PHONE_NUMBER:
     description: 'Phone number in your Twilio account to send the SMS from'
     required: true
-  toPhoneNumber:
+  TO_PHONE_NUMBER:
```
Comment (bnb) [inline action.yml:8]: you should check the README PR since I think it refers to `fromPhoneNumber` and `toPhoneNumber`.

File: action.yml [hunk 2]
```diff
@@ -14,11 +14,8 @@ inputs:
   TWILIO_ACCOUNT_SID:
     description: 'A Twilio Account SID. Can alternatively be stored in environment'
     required: true
-  TWILIO_API_KEY:
-    description: 'A Twilio API Key. Can alternatively be stored in environment'
-    required: true
-  TWILIO_API_SECRET:
-    description: 'A Twilio API Secret. Can alternatively be stored in environment'
+  TWILIO_AUTH_TOKEN:
+    description: 'A Twilio Auth Token. Can alternatively be stored in environment'
```
Comment (bnb) [inline action.yml:18]: We should remove `Can alternatively be stored in environment` from all of these because it's... obvious/how it works.

**Comment by bnb** [comment]
Should also be merged with #72

**Review by bnb — APPROVED** [review]

**Comment by ayyrickay** [comment]
@sbrij001 It looks like this can now be merged in

**Pull request merged**